package/gluon-core/luasrc/lib/gluon/upgrade/140-firewall-rules

```diff
@@ -11,14 +9,51 @@ local function reject_input_on_wan(zone)
 
 	return true
 end
+
 uci:foreach('firewall', 'zone', reject_input_on_wan)
 
-uci:section('firewall', 'rule', 'wan_ssh', {
-	name = 'wan_ssh',
-	src = 'wan',
-	dest_port = '22',
-	proto = 'tcp',
-	target = 'ACCEPT',
+-- the client zone is set up by gluon-client-bridge
+--
+uci:section('firewall', 'zone', 'mesh', {
+	name = 'mesh',
```

Comment:
I think adding `network = {}` to remove any potential old values and get a known state for other scripts to work on would be nice to make this more robust an future-proof.

Reply:
done.